metrics/cli/lib/doctor/doctor.dart

```diff
@@ -56,7 +53,7 @@ class Doctor {
     await _checkVersion(_gcloudService.version);
     await _checkVersion(_npmService.version);
     await _checkVersion(_gitService.version);
-    await _checkVersion(_firebaseCommand.version);
+    await _checkVersion(_firebaseService.version);
```

Comment:
It would be great to update the `_checkVersion` method to take an `InfoService` as an argument and check its version.